server/eb/README.md

```diff
@@ -58,17 +59,26 @@ There are many more options to these commands that may be important or necessary
    is the location where the matrix files are located. 
    This environment variable will override the dataroot in the config file (if specified).
    
-   - Note:  Certain features, such as diffexp and user annotations, are automatically disabled by the EB app,
+   - Note:  Certain features, such as user annotations, are automatically disabled by the EB app,
    and cannot be enabled using configuration.  They may be enabled manually by modifying app.py, however
    this is not supported or recommended at this time.
    
-4. Prepare static endpoints
+4. Customization
 
-   The cellxgene server can server additional static webpages that may be associated with the deployment.
-   These include the about_legal_tos (terms of service), and bout_legal_privacy, for example.  
-   To use this feature, do the following:
+The deployment can be customized in several ways, by adding files to a directory called
+`customize` which is placed in this directory.
+ 
+config file: 
 
-   * In this directory, create a sub directory called "static".  
+This was described in the previous section.  
+
+static files: 
+
+The cellxgene server can server additional static webpages that may be associated with the deployment.
```

Comment:
s/can server/can serve/

Reply:
s/that may be associated with the deployment/that will be deployed with the app/